packages/components/src/Form/Form.test.tsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { cleanup, fireEvent, render, waitFor } from "@testing-library/react";
+import { Form } from ".";
+import { InputText } from "../InputText";
+
+afterEach(cleanup);
+
+test("calls the submit handler if the form is valid", async done => {
+  const submitHandler = jest.fn();
+  const { getByText, getByLabelText } = render(
+    <MockForm onSubmit={submitHandler} />,
+  );
+
+  const input = getByLabelText("test form");
+  fireEvent.change(input, { target: { value: "hello" } });
+  fireEvent.click(getByText("submit"));
+
+  waitFor(() => {
```

Comment:
does `await waitFor` work here instead of the `done()`?

Reply:
we do need the `done()` here, but it can be taken out of the `waitFor`. I'm not sure if it makes a difference or not?